test/integration/runner_task_test.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+require 'test_helper'
+
+require 'kubernetes-deploy/runner_task'
+class RunnerTaskTest < KubernetesDeploy::IntegrationTest
+  def test_works
+    deploy_fixtures("hello-cloud")
```

Comment:
Acutally, don't deploy the whole thing. It'll be way slower than necessary. Use `subset: [filename]` kwarg to select just the configmap and podtemplate you need. Still do add a helper to `FixtureSetAssertions::HelloCloud` (and call it from `assert_all_up`, since this is now part of that deploy set) and call that + the existing one for the configmap here.